helm/application-namespace-prep/templates/authenticator-rolebinding.yml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.authnRoleBinding.create -}}
+---
+{{- $g := (lookup "v1" "ConfigMap" .Values.authnK8s.namespace .Values.authnK8s.goldenConfigmap).data }}
```

Comment:
`goldenConfigmap` should be `goldenConfigMap`, otherwise lookup fails with the proper value keys.